tests/test_data/onesweetappetite.com/onesweetappetite.json

```diff
@@ -0,0 +1,71 @@
+{
+  "author": "Jesseca",
+  "canonical_url": "https://onesweetappetite.com/ranch-chicken/",
+  "site_name": "One Sweet Appetite",
+  "host": "onesweetappetite.com",
+  "language": "en-US",
+  "title": "Ranch Chicken",
+  "ingredient_groups": [
+    {
+      "ingredients": [
+        "4 chicken breast halves",
+        "1/2 teaspoon salt",
+        "1/2 teaspoon garlic powder",
+        "1/4 teaspoon pepper",
+        "1/4 cup ranch dressing",
+        "1/2 cup breadcrumbs",
+        "3 tablespoons butter (melted)"
+      ],
+      "purpose": null
+    }
+  ],
+  "ingredients": [
+    "4 chicken breast halves",
+    "1/2 teaspoon salt",
+    "1/2 teaspoon garlic powder",
+    "1/4 teaspoon pepper",
+    "1/4 cup ranch dressing",
+    "1/2 cup breadcrumbs",
+    "3 tablespoons butter (melted)"
+  ],
+  "instructions": "Preheat the oven to 425 degrees.\nMist a baking tray or 9x13 pan with cooking spray. Set aside.\nSlice the chicken in half lengthwise, or pound to 1/2 inch thickens.\nPlace on the prepared baking tray, making sure not to overlap.\nSeason the chicken with salt, pepper, and garlic powder.\nBrush 1 tablespoon of ranch dressing onto the top of each chicken piece.\nMix the bread crumbs with the butter. Spoon over the top of each chicken.\nBake 15 minutes, or until the chicken reaches an internal temperature of 165 degrees.\nServe with extra ranch dressing if desired.",
+  "instructions_list": [
+    "Preheat the oven to 425 degrees.",
+    "Mist a baking tray or 9x13 pan with cooking spray. Set aside.",
+    "Slice the chicken in half lengthwise, or pound to 1/2 inch thickens.",
+    "Place on the prepared baking tray, making sure not to overlap.",
+    "Season the chicken with salt, pepper, and garlic powder.",
+    "Brush 1 tablespoon of ranch dressing onto the top of each chicken piece.",
+    "Mix the bread crumbs with the butter. Spoon over the top of each chicken.",
+    "Bake 15 minutes, or until the chicken reaches an internal temperature of 165 degrees.",
+    "Serve with extra ranch dressing if desired."
+  ],
+  "category": "Dinner Recipes",
+  "yields": "4 servings",
+  "description": "This easy ranch chicken recipe is the perfect busy weeknight dinner! Made with only a handful of ingredients and a family favorite recipe!",
+  "total_time": 20,
+  "cook_time": 15,
+  "prep_time": 5,
+  "cuisine": "American",
+  "ratings": 4.83,
+  "ratings_count": 0,
```

Comment:
Hm.  This seems odd - why is this zero when there is a non-zero aggregate rating count in the `testhtml`?

Reply:
Turns out the ratings_count test logic was backwards, swapping the assert showed a bunch of failing tests. just pushed a fix

Reply:
Hrm.  Perhaps we should change it to an equality check?  (simpler to reason about, even if it does mean that we may have to update the relevant JSON values more often)

Reply:
I agree, originally seemed like a cool idea but I think the extra test logic is overcomplicating things.